site/gatsby-site/src/tailwind.css

```diff
@@ -558,4 +558,40 @@
     .bootstrap .list-group-item+.list-group-item {
         border-top-width: 0;
     }
+    .tsne-visualization-wrapper>.react-transform-wrapper {
+        @apply w-full h-full !important;
+    }
+
+    .tsne-visualization-wrapper>.react-transform-wrapper>.react-transform-component {
+        @apply h-[calc(100vh-2rem)] w-[calc(100vh-2rem)] overflow-visible !important;
+    }
+
+    .tsne-visualization-item>a:not(.incident-card) {
+        @apply text-inherit absolute text-[75%] h-[2em] w-[2em] px-[2px] rounded-[50%] -ml-4 -mt-4 z-2 text-center leading-8 shadow-tsne flex justify-center items-center;
+    }
+
+    .tsne-visualization-item .incident-card {
+        @apply block min-h-[15em] w-60 rounded-[0.25rem] shadow-tsne absolute p-4 overflow-hidden;
```

Comment:
![image](https://user-images.githubusercontent.com/25443411/227065327-9b6519d9-ad1c-4bc0-aa16-fdd02946e4a3.png)

I think this also needs `bg-white`.